src/pgduckdb_duckdb.cpp

```diff
@@ -151,9 +162,14 @@ DuckdbCreateConnection(List *rtables, PlannerInfo *planner_info, List *needed_co
 	auto &db = DuckDBManager::Get().GetDatabase();
 	/* Add DuckDB replacement scan to read PG data */
 	auto con = duckdb::make_uniq<duckdb::Connection>(db);
-	con->context->registered_state->Insert(
-	    "postgres_scan", duckdb::make_shared_ptr<PostgresReplacementScanDataClientContextState>(rtables, planner_info,
-	                                                                                            needed_columns, query));
+	auto &context = *con->context;
+
+	context.registered_state->Insert(
+	    "postgres_state", duckdb::make_shared_ptr<PostgresContextState>(rtables, planner_info, needed_columns, query));
+	auto res = context.Query("set search_path='pgduckdb.main'", false);
```

Comment:
Is this line still needed now?

Reply:
At least searching the `main` schema by default isn't necessary anymore I think. Maybe change it to:
```suggestion
	auto res = context.Query("USE pgduckdb", false);
```

Reply:
Side-note: I think in a future PR we might want to change the the vendored in ruleutils to include the `pgduckdb` catalog in the fully table name in a query i.e. `pgduckdb.public.t` instead of `public.t`. But that can wait until we actually need it, e.g. to avoid ambiguities once we support having multiple databases open in duckdb.

Reply:
okay, seems reasonable to me. Resolving this then.